tools/daemonUtil.js

```diff
@@ -0,0 +1,94 @@
+const forever = require('forever-monitor');
```

Comment:
nit: how about just add the .Monitor here, and then below we can just do a normal new Monitor(stuff)